app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(text: params[:text], author_id: current_user.id, post_id: @post.id)
+
+    respond_to do |format|
+      format.html do
+        if @comment.save
+          redirect_to user_post_path(@post.author.id, @post.id), notice: 'Comment created successfully'
+        else
+          redirect_to user_post_path(@post.author.id, @post.id), alert: 'Comment not created, try again!'
+        end
+      end
+    end
+  end
+end
```

Comment:
- [ ] You are currently missing the comment#new action. Let's add this action method for new comments.

Reply:
Thank you for letting me know. I did it. 👍